containers/nginx-storage/install-dependencies.sh.j2

```diff
@@ -15,7 +15,9 @@ opm get spacewander/luafilesystem
 # Create .so symlink for libcrypto.so manually.
 # In normal circumstances the link would be created by ldconfig,
 # but in Alpine Linux ldconfig is non-functional.
-ln -s /usr/lib/libcrypto.so.41 /usr/lib/libcrypto.so
+# WARNING !!! if you get error "<Error loading shared library libcrypto.so: No such file or directory>",
+# you need to update the version number in libcrypto.so.XX
```

Comment:
The `<>` was only to show that it's a place holder :) I did not mean to include it in the message :)